sencha-workspace/packages/slate-cbl/src/widget/AttachmentsField.js

```diff
@@ -42,6 +46,29 @@ Ext.define('Slate.cbl.widget.AttachmentsField', {
         disabled: true
     }],
 
+    updateReadOnly: function(readOnly) {
+        if (!this.rendered) {
+            return this.on('render', function() {
+                return this.updateReadOnly(readOnly);
+            });
```

Comment:
@nbey be sure to pass the `{ single: true }` option with this `.on()` call, or this component being rendered within something like a tab panel or window that can be rendered/unrendered several times will result in an infinitely growing stack of this function getting called for each update